.gitignore

```diff
@@ -1,3 +1,7 @@
 /node_modules/
 /TODO
 /NO_GIT/
+/lib/libWebRTC-LATEST-Universal-Release.a
+/lib/.DS_Store
+.DS_Store
+
```

Comment:
Please remove these lines from the .gitignore. OS's specific files can be globally ignored by creating your own `HOME/.gitignore_global` file.